setup.php

```diff
@@ -28,7 +28,7 @@
  * -------------------------------------------------------------------------
  */
 
-define ('PLUGIN_UNINSTALL_VERSION', '2.8.0');
+define ('PLUGIN_UNINSTALL_VERSION', '2.8.1');
```

Comment:
Will be do during release process.

```suggestion
define ('PLUGIN_UNINSTALL_VERSION', '2.8.0');
```